javatests/arcs/android/storage/handle/AndroidHandleManagerTest.kt

```diff
@@ -184,14 +184,9 @@ class AndroidHandleManagerTest : LifecycleOwner {
         assertThat(readBack.isAlive(coroutineContext)).isFalse()
         assertThat(readBack.isDead(coroutineContext)).isTrue()
 
-        // Stash entity1 in the RamDisk manually, so it becomes alive.
-        RamDisk.memory[readBack.storageKey.childKeyWithComponent(readBack.id)] = VolatileEntry(
-            CrdtEntity.Data(VersionMap("foo" to 1), entity1) {
-                if (it is Reference) it
-                else CrdtEntity.Reference.buildReference(it)
-            },
-            0
-        )
```

Comment:
Are there other reference liveness paths that we should check? Maybe the two-store model, for example.

Reply:
What would the two-store model involve?

Reply:
Creating two independent HandleManagers and making sure references work properly across them as well (since there are two stores, the updates would come via the driver).

Reply:
I have another PR setting up some tests like this, so I'll just finish that up after this is merged.